### PR TITLE
Make review command case insensitive

### LINKED ISSUE #303: [PE-D][Tester D] does not accept L to label as easy although image seems to suggest so
image on left suggest pressing L to tag as easy 
perhaps can accept both upper and lower case L

-------------
Labels: `type.FeatureFlaw` `severity.Low`
original: rachtan27/ped#8

### PR DESCRIPTION
**Pull request by HJ-PRK**
Closes #303 #271

### CODE REVIEW

**Review by junianob — COMMENTED** [review]
I think you should call a util (see ParserUtil.java) to perform lowercase operations on the command rather than doing it directly on MasterDeckParser.java 
Let me know what you think!

File: src/main/java/seedu/address/logic/parser/MasterDeckParser.java [hunk 1]
```diff
@@ -208,6 +208,23 @@ public Command parseCommandInMainSelectedMode(String userInput) throws ParseExce
         case FindCardsCommand.COMMAND_WORD:
             return new FindCardsCommandParser().parse(arguments);
 
+        // Other Commands
```
Comment (ruiyigan) [inline src/main/java/seedu/address/logic/parser/MasterDeckParser.java:211]: just curious why did you shift Other Commands to be in front of Review-related Commands?

**Review by ruiyigan — DISMISSED** [review]
hmm can you check if your implementation works for `endReview`? Because the command word for EndReviewCommand is `endReview` and if you make the command world lowercase the user may not be able to end review.

**Comment by HJ-PRK** [comment]
> hmm can you check if your implementation works for `endReview`? Because the command word for EndReviewCommand is `endReview` and if you make the command world lowercase the user may not be able to end review.

You're right. I'm going to fix it.

File: src/main/java/seedu/address/logic/parser/MasterDeckParser.java (hunk 1 above)
Comment (HJ-PRK) [reply src/main/java/seedu/address/logic/parser/MasterDeckParser.java:211]: No reason actually. I'm going to shift it to be after Review-related commands now :>

**Review by junianob — APPROVED** [review]
Lgtm

**Pull request merged**